### ISSUE TITLE
Topics to open

### ISSUE THREAD

**Issue by ajschumacher** [ajschumacher/ajschumacher.github.io#226]
Things that maybe I should learn more about and/or write explainers for...

**Comment by ajschumacher** [comment]
Heckman correction

**Comment by ajschumacher** [comment]
the kernel trick (formerly #221)

**Comment by ajschumacher** [comment]
wavelets (formerly #210)

> "... much more modern ideas, like the theory of wavelets developed in the 1970s and '80s by Jean Morlet, Stéphane Mallat, Yves Meyer, Ingrid Daubechies, and others" (page 330, Ellenberg)

**Comment by ajschumacher** [comment]
instrumental variables (formerly #201)

The Book of Why has some stuff; pages 248-249 etc.

**Comment by ajschumacher** [comment]
Turbocodes (formerly #200)

**Comment by ajschumacher** [comment]
support vector machines (formerly #196)

**Comment by ajschumacher** [comment]
Baum–Welch algorithm (formerly #187)

https://en.wikipedia.org/wiki/Baum%E2%80%93Welch_algorithm

(recommended via that book about Simons)

**Comment by ajschumacher** [comment]
Fast Fourier transform (FFT) (formerly #154)

Downey's stuff might be a good start... Anyway I saw this tweet:

 * https://twitter.com/allendowney/status/931172574628843520

Which links to this notebook:

 * https://github.com/AllenDowney/ThinkComplexity2/blob/master/code/fftsoln.ipynb

---

https://twitter.com/jezzamonn/status/1083498750050553856

mention Fourier in case I search by that...

https://www.jezzamon.com/fourier/index.html

---

FFT comes up here: https://www.quantamagazine.org/mathematicians-discover-the-perfect-way-to-multiply-20190411/ kind of surprisingly?


---

https://realpython.com/python-scipy-fft/


---

This has a FFT section: https://pythonnumericalmethods.berkeley.edu/notebooks/Index.html

---

twiddle factors?


---

> The mathematical operation of convolution has a long and rich history dating back
to famous mathematicians such as Euler, Laplace, and Cauchy in the late 1700’s
and early 1800’s. In 1822, Fourier discovered the Fourier transform that converts
a signal from the time domain to the frequency domain and proved the theorem
that multiplication in frequency is equivalent to convolution in time. The Fourier
transform forms the basis for modern time series analysis in the frequency domain.
In the middle of the 20th century, convolution found its first practical application
in signal processing of audio and images, combining two functions to produce a
third function that represents how one function modifies the other. In 1965, Cooley
and Tukey developed the fast Fourier transform (FFT) which revolutionized signal
processing since it improved computational efficiency for processing signals (this is
the same Tukey who founded exploratory data analysis presented in Chapter 1). In
the context of traditional computer vision, convolution was used to apply filters to
images for tasks such as edge detection, blurring, and sharpening. A different type of
filter is used for each type of task; for example, a Sobel filter is used for edge detection.
Convolution found its way into neural networks in the late 1980s and early 1990s,
with the development of convolutional neural networks (CNNs) by researchers such
as Yann LeCun. The seminal work by LeCun et al. in 1998 on the LeNet architecture
demonstrated the effectiveness of CNNs for handwritten digit recognition. LeCun
laid the groundwork for the deep learning revolution that would follow decades l

**Comment by ajschumacher** [comment]
Hungarian association algorithm (formerly #153)

**Comment by ajschumacher** [comment]
particle filters (formerly #151)

**Comment by ajschumacher** [comment]
transducers (formerly #150)

**Comment by ajschumacher** [comment]
Kalman filters (formerly #149)

Least-squares estimation: from Gauss to Kalman https://www2.pv.infn.it/~fontana/download/lect/Sorenson.pdf

I had some old note like this:

> Wiener process / Kalman filtering (connection with Gaussian process?)

Fun demo recommended by Gerardo Leal: https://www.cs.utexas.edu/~teammco/misc/kalman_filter/

**Comment by ajschumacher** [comment]
IRR function? in Excel? (formerly #131)

**Comment by ajschumacher** [comment]
beam search / viterbi (formerly #109)

**Comment by ajschumacher** [comment]
`top`, `htop`, etc. (formerly #101)

**Comment by ajschumacher** [comment]
softmax temperature (formerly #105)

**Comment by ajschumacher** [comment]
inverting / pseudo-inverting a matrix (formerly #82)

fun highlights of various intuitions: https://twitter.com/michael_nielsen/status/1073673971583705089

**Comment by ajschumacher** [comment]
tSNE/UMAP (formerly #41)

**Comment by ajschumacher** [comment]
"Relative Weight Analysis"?

This fairly bad post has gotten some play online:

https://towardsdatascience.com/key-driver-analysis-in-python-788beb9b8a7d

It seems to be largely based on this better article:

https://www.researchgate.net/profile/Nicola_Di_Girolamo2/post/Which_is_the_Best_Effect-Size_for_Predictors_in_Multivariate_Logistic_Regression/attachment/59d632e0c49f478072ea1d4a/AS%3A273637431545864%401442251655932/download/Relative+Importance+Analysis%3A+A+Useful+Supplement+to+Regression+Analysis.pdf

Kind of neat as another kind of feature importance?

**Comment by ajschumacher** [comment]
Learn more languages? C++? Elixir?

**Comment by ajschumacher** [comment]
figure out a good way to do speech-to-text? for calls and so on?

**Comment by ajschumacher** [comment]
Morris traversal?

https://www.educative.io/edpresso/what-is-morris-traversal

https://m.youtube.com/watch?v=3rdM5fBjnB8

**Comment by ajschumacher** [comment]
https://en.wikipedia.org/wiki/Toeplitz_matrix for convolution

**Comment by ajschumacher** [comment]
Power law: scaling to distribution?
